azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/controlbar/ControlBarView.kt

```diff
@@ -91,10 +129,12 @@ internal class ControlBarView : ConstraintLayout {
             CameraOperationalStatus.ON -> {
                 cameraToggle.isSelected = true
                 cameraToggle.isEnabled = permissionIsNotDenied
+                cameraToggle.contentDescription = context.getString(R.string.azure_communication_ui_calling_setup_view_button_video_on)
```

Comment:
Hey. Have you tried using announceForAccessibility(String) method in this state update block? If that solves the issue it is probably much cleaner way to solve the issue.